src/appleseed.cli/stdouttilecallback.h

```diff
@@ -38,11 +38,17 @@
 namespace appleseed {
 namespace cli {
 
+enum TileOutputOptions
```

Comment:
Exactly. Can you just move it inside `StdOutTileCallbackFactory` and use C++11's scoped enum?

Reply:
Yes!